addons/google_calendar/models/google_calendar.py

```diff
@@ -589,153 +579,141 @@ def synchronize_events(self, cr, uid, ids, lastSync=True, context=None):
             "url": ''
         }
 
-    def create_new_events(self, cr, uid, context=None):
-        if context is None:
-            context = {}
-
+    def create_new_events(self):
         new_ids = []
-        ev_obj = self.pool['calendar.event']
-        att_obj = self.pool['calendar.attendee']
-        user_obj = self.pool['res.users']
-        myPartnerID = user_obj.browse(cr, uid, uid, context=context).partner_id.id
-
-        context_norecurrent = context.copy()
-        context_norecurrent['virtual_id'] = False
-        my_att_ids = att_obj.search(cr, uid, [('partner_id', '=', myPartnerID),
+        CalendarAttendenee = self.env['calendar.attendee']
+        my_partner_id = self.env.user.partner_id.id
+
+        my_attendees = CalendarAttendenee.with_context(virtual_id=False).search([('partner_id', '=', my_partner_id),
                                     ('google_internal_event_id', '=', False),
                                     '|',
-                                    ('event_id.stop', '>', self.get_minTime(cr, uid, context=context).strftime(DEFAULT_SERVER_DATETIME_FORMAT)),
-                                    ('event_id.final_date', '>', self.get_minTime(cr, uid, context=context).strftime(DEFAULT_SERVER_DATETIME_FORMAT)),
-                                    ], context=context_norecurrent)
-        for att in att_obj.browse(cr, uid, my_att_ids, context=context):
-            if not att.event_id.recurrent_id or att.event_id.recurrent_id == 0:
-                st, response, ask_time = self.create_an_event(cr, uid, att.event_id, context=context)
-                if status_response(st):
-                    update_date = datetime.strptime(response['updated'], "%Y-%m-%dT%H:%M:%S.%fz")
-                    ev_obj.write(cr, uid, att.event_id.id, {'oe_update_date': update_date})
-                    new_ids.append(response['id'])
-                    att_obj.write(cr, uid, [att.id for att in att.event_id.attendee_ids], {'google_internal_event_id': response['id'], 'oe_synchro_date': update_date})
-                    cr.commit()
-                else:
-                    _logger.warning("Impossible to create event %s. [%s] Enable DEBUG for response detail.", att.event_id.id, st)
-                    _logger.debug("Response : %s" % response)
+                                    ('event_id.stop', '>', self.get_minTime().strftime(DEFAULT_SERVER_DATETIME_FORMAT)),
+                                    ('event_id.final_date', '>', self.get_minTime().strftime(DEFAULT_SERVER_DATETIME_FORMAT)),
+                                    '|',
+                                    ('event_id.recurrent_id', '=', False),
+                                    ('event_id.recurrent_id', '=', 0)
+                                    ])
+        for att in my_attendees:
```

Comment:
It seems this method missing some code if compared to code in current <a href="https://github.com/odoo/odoo/blob/master/addons/google_calendar/google_calendar.py#L613">master</a>(stuff related to other_google_ids). Would you please check it.